extensions/cloudflare/src/view-pages.tsx

```diff
@@ -0,0 +1,353 @@
+import { Action, ActionPanel, Detail, Icon, List } from '@raycast/api';
+import { useEffect, useState } from 'react';
+
+import Service, { Deployment, Domain, Page } from './service';
+import {
+  getCommitUrl,
+  getDeploymentStatusIcon,
+  getDeploymentUrl,
+  getDomainStatusIcon,
+  getEmail,
+  getKey,
+  getPageUrl,
+  getRepoUrl,
+  handleNetworkError,
+  toUrl,
+} from './utils';
+
+const service = new Service(getEmail(), getKey());
+
+function Command() {
+  const [pages, setPages] = useState<Page[]>([]);
+  const [accountId, setAccountId] = useState<string>('');
+  const [isLoading, setLoading] = useState(true);
+
+  useEffect(() => {
+    async function fetchPages() {
+      try {
+        const accounts = await service.listAccounts();
+        if (accounts.length === 0) {
+          setPages([]);
+          setLoading(false);
+        }
+        const account = accounts[0];
```

Comment:
Out of curiosity, could there be ore than 1 account? and in that case would it make sense to list the pages for all of them (maybe using `List.Section`)?

Reply:
I think that might be the case, will update to handle it.

Reply:
Fixed in c99bb3fabb990ecea3fbd556e9d6a44a0473a66d.